scripts/bfasst/transform/xilinx_phys_netlist.py

```diff
@@ -0,0 +1,161 @@
+""" Creates a xilinx netlist that has only physical primitives"""
+
+
+import re
+import subprocess
+import jpype
+import jpype.imports
+
+from bfasst.config import VIVADO_BIN_PATH
+from bfasst.paths import THIRD_PARTY_PATH
+from bfasst.status import Status, TransformStatus
+from bfasst.transform.base import TransformTool
+
+jpype.startJVM(classpath=[str(THIRD_PARTY_PATH / "rapidwright-2022.2.1-standalone-lin64.jar")])
```

Comment:
Can we do:
```
try:
    import rapidwright  # This starts jpype, and works if you have only run `pip install rapidwright`
except ImportError:
    jpype.startJVM(classpath=[str(THIRD_PARTY_PATH / "rapidwright-2022.2.1-standalone-lin64.jar")])
```

Reply:
If we add rapidwright to the requirements.txt, then you can just run `import rapidwright` to start JPype and then you can do your com.xilinx.rapidwright imports. Unless you are trying to use a local, modified version of rapidwright with python?

Reply:
Any reason for the `try`?  If we are going to install the pip package, shouldn't we just assume it exists?

Reply:
No reason for the try in that case. Sorry I wrote this comment first before the other ones. Yes, we can just do the import rapidwright.

Reply:
Note that with pip install rapidwright, we effectively just get the jar, so if we want the interchange project as well, I think we'd need to download that separately.

Reply:
I've updated to use our existing rapidwright code:
```python
jpype.startJVM(
    classpath=[
        str(THIRD_PARTY_PATH / "RapidWright" / "bin"),
        *(str(s) for s in (THIRD_PARTY_PATH / "RapidWright" / "jars").glob("*.jar")),
    ]
)
```